shared/src/main/scala/coop/rchain/lmdb/LMDBStore.scala

```diff
@@ -28,14 +34,18 @@ final case class LMDBStore[F[_]: Sync](env: Env[ByteBuffer], dbi: Dbi[ByteBuffer
     }
 
   def withReadTxnF[R](f: Txn[ByteBuffer] => R): F[R] =
-    Sync[F].delay {
-      withTxn(env.txnRead)(f)
-    }
+    Sync[F]
+      .delay {
+        withTxn(env.txnRead)(f)
+      }
+      .timer("lmdb.read")
```

Comment:
Upon each invocation of this method the underlying `TrieMap` is going to be accessed to search for the timer if it exists. I'm worried about the performance impact. This should be only a temporary change for a single debugging session, I really don't think this should go into dev

Reply:
removed all LMDB-Store metrics